### PR TITLE
change ctx.send_help to ctx.invoke(help_command)

### LINKED ISSUE #624: Using group commands without specifying a subcommand sends the default help command.
## Description
Whenever someone uses a group command without providing a subcommand, it sends the default help command instead of the one we already have.:

## Steps to Reproduce
If I just do `.group_name`, it sends the default help command.

## Expected Behaviour
It should use our help command, so it should look like this:
image

## Actual Behaviour
It sends this:
image

## Known Impacted Platforms

- [x] Web
- [x] Desktop
- [x] Android App
- [x] iOS App

## Possible Solutions
We are currently using [Context.send_help](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Context.send_help), but our help command isn't a [commands.HelpCommand](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#helpcommand) so it sends the default one. Instead, we should invoke the help command instead using [Context.invoke](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Context.invoke) to invoke our help command.

## Would you like to implement a fix?

***Note: For high-priority or critical bugs, fixes may be implemented by staff.***

- [x] I'd like to implement the bug fix
- [ ] Anyone can implement the bug fix

### PR DESCRIPTION
**Pull request by ToxicKidz**
## Relevant Issues
Closes #624 

## Description
I've changed all the `ctx.send_help`'s to `bot.get_command("help")` and `ctx.invoke(help_command, ctx.command.name)`.

## Reasoning

It looks a lot better.

## Screenshots
Before:
image

After:
image

## Additional Details
Affected commands are `.games`, `.snakes`, `.space`,  `.emoji`, `.minesweeper`, `.movies`, `.status`, `.extensions` (and its subcommands), and `.bemyvalentine`.

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] If dependencies have been added or updated, run `pipenv lock`?
- [x] **Lint your code** (`pipenv run lint`)?
- [x] Set the PR to **allow edits from contributors**?

### CODE REVIEW

**Review by Shivansh-007 — COMMENTED** [review]
Can we add `help_command` a class attribute in `Bot` class in `bot/bot.py`, then we could just do:
```py
await ctx.invoke(self.bot.help_command, ctx.command.name)
```

**Comment by anand2312** [comment]
> Can we add `help_command` a class attribute in `Bot` class in `bot/bot.py`, then we could just do:
> 
> ```python
> await ctx.invoke(self.bot.help_command, ctx.command.name)
> ```

`bot.help_command` is an [attribute of `commands.Bot`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.help_command)  and has to be of type `HelpCommand`. I don't know how overriding this with something that is not of that type will work.

**Comment by Shivansh-007** [comment]
> > Can we add `help_command` a class attribute in `Bot` class in `bot/bot.py`, then we could just do:
> > ```python
> > await ctx.invoke(self.bot.help_command, ctx.command.name)
> > ```
> 
> `bot.help_command` is an [attribute of `commands.Bot`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.help_command) and has to be of type `HelpCommand`. I don't know how overriding this with something that is not of that type will work.

Ah right! Forgot about the `help_command`, didn't meant to override, what I meant was to add another class attribute to the `Bot` class, so we won't need to do 
```py
help_command = self.bot.get_command("help")
```
everytime we are sending help on a group command.

**Review by Akarys42 — APPROVED** [review]
LGTM, thanks a lot!

File: bot/utils/extensions.py [hunk 1]
```diff
@@ -31,4 +33,12 @@ def on_error(name: str) -> NoReturn:
         yield module.name
 
 
+async def invoke_help_command(ctx: Context, *commands: str) -> None:
```
Comment (ks129) [inline bot/utils/extensions.py:36]: You can receive full name with parents using `ctx.command.qualified_name`, so you don't have to pass command name manually.

**Review by ks129 — CHANGES_REQUESTED** [review]

**Review by ChrisLovering — APPROVED** [review]
Looks good to me!

**Review by ks129 — APPROVED** [review]

**Pull request merged**